modules/core/src/controllers/controller.js

```diff
@@ -99,6 +101,12 @@ export default class Controller {
         return this._onPinch(event);
       case 'pinchend':
         return this._onPinchEnd(event);
+      case 'tripanstart':
+        return this._onTriplePanStart(event);
```

Comment:
Total nit: These per-event methods are nice but as the set of events grows this generates a lot of non-minifiable method names (compared to e.g. `onEvent` `switch(event.type)` statement).